draft-ietf-scitt-architecture.md

```diff
@@ -331,12 +331,15 @@ Before an Issuer is able to produce Signed Statements, it must first create its
 A DID can be *resolved* into a *key manifest* (a list of public keys indexed by a *key identifier*) using many different DID methods.
 
 Issuers MAY choose the DID method they prefer, but with no guarantee that all Transparency Services will be able to register their Signed Statements.
-To facilitate interoperability, all Transparency Service implementations SHOULD support the `did:web` method {{DID-WEB}}.
+To facilitate interoperability, all Transparency Service implementations MUST support the `did:web` method {{DID-WEB}}.
 For instance, if the Issuer publishes its manifest at `https://sample.issuer/user/alice/did.json`, the DID of the Issuer is `did:web:sample.issuer:user:alice`.
 
 Issuers SHOULD use consistent decentralized identifiers for all their Statements about Artifacts, to simplify authorization by Verifiers and auditing.
-They MAY update their DID manifest, for instance to refresh their signing keys or algorithms, but they SHOULD NOT remove or change any prior keys unless they intend to revoke all Signed Statements that are registered as Transparent Statements issued with those keys.
-This DID appears in the Issuer protected header of Signed Statements' Envelopes, while the version of the key from the manifest used to sign the Signed Statement is written in the `kid` header.
+If an issuer uses multiple DIDs (for instance, because their clients support different resolution methods), they MUST ensure that statements signed under each DID are consistent.
```

Comment:
The verifier is interested in the authenticity of the issuer of the transparent statement and signed statement... The verifier trusts the TS to vet the issuer of the signed statement... the verifier benefits from the transparency service, if they can't trust the TS, there is not reason to do SCITT.

Reply:
Tracked in https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/issues/99